modules/update-restricted.adoc

```diff
@@ -29,9 +29,9 @@ If you have a local OpenShift Update Service, you can update by using the connec
 +
 [source,terminal]
 ----
-$ oc adm upgrade --allow-explicit-upgrade --to-image ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}<sha256_sum_value> <1>
+$ oc adm upgrade --allow-explicit-upgrade --to-image ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}@<digest> <1>
 ----
-<1> The `<sha256_sum_value>` value is the sha256 sum value for the release from the image signature ConfigMap, for example, `@sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92`
+<1> The `<digest>` value is the sha256 digest for the release from the image signature ConfigMap, for example, `@sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92`
```

Comment:
`@` isn't part of [the digest][1], just `sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92`.  Can we drop the `@` from this line, now that it's inlined in the template two lines earlier?

[1]: https://github.com/opencontainers/image-spec/blob/v1.0.2/descriptor.md#digests

Reply:
Missed that detail while making the other changes, thanks!